apps/trading/pages/_app.page.tsx

```diff
@@ -25,47 +25,49 @@ function AppBody({ Component, pageProps }: AppProps) {
   const store = useGlobalStore();
   const { VEGA_NETWORKS } = useEnvironment();
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
chore (non-blocking): i think the comment disabling the lint can also be removed, if the theme variable is used

Reply:
Nice, thanks Botond